utils/eliteDangerousAPI.js

```diff
@@ -45,18 +62,33 @@ function getEliteStatus() {
 function getGalnetFeed() {
   return new Promise(async (fulfill, reject) => {
     try {
-      let url = "https://www.alpha-orbital.com/galnet-feed";
-      fulfill(JSON.parse(await request(url)));
+      axios.get("https://www.alpha-orbital.com/galnet-feed")
+        .then(function (response) {
+          fulfill(JSON.parse(response.data));
+        }).catch(function (error) {
+          reject(error);
+        });
     } catch (error) { reject(error); }
   });
 }
 
-function getObjectFromAPI(params) {
+function getObjectFromAPIAxios(url, params) {
   return new Promise(async (fulfill, reject) => {
     try {
-      let url = apiUrl + params;
-      var text = await request(url);
-      fulfill(JSON.parse(text));
+      axios.get(url, {
+        params: params
+      })
+      .then(function (response) {
+        if (response.statusCode.toString()[0] == "2") {
+          fulfill(JSON.parse(response.data));
+        } else {
+          reject("Non-OK Status Code From API: " + response.statusCode.toString());
+        }

```

Comment:
It will already reject on a non-200 response. Additionally, `response.data` comes pre-parsed.
```suggestion
        fulfill(response.data);

```